### PR TITLE
Proofread flank blog

### PR DESCRIPTION
**Pull request by EmmaKiwi**

### CODE REVIEW

File: content/posts/2021-04-19-flank.md [hunk 1]
```diff
@@ -10,33 +10,33 @@ I will soon give a live presentation about Flank
...
+In order to fully prepare, I've decided to write a blog post about
+my experience using this wonderful tool Flank, which both reduces the waiting time for running UI tests
+and offers new opportunities, thanks to the way it was implemented.
 
 # Test Lab
 
 If you're already using Android E2E (end-to-end, aka UI tests) tests,
 you already probably know Firebase Test Lab.
 
-It's a Google service allowing to access on-demand virtual or real devices,
```
Comment (EmmaKiwi) [inline content/posts/2021-04-19-flank.md:22]: `allowing to` -> Please stop using this phrase 🤣

**Comment by EmmaKiwi** [comment]
The formatting may be off 😕 I was a little unsure about how to edit it

File: content/posts/2021-04-19-flank.md (hunk 1 above)
Comment (pgreze) [reply content/posts/2021-04-19-flank.md:22]: @EmmaKiwi but I like it so much 😆

**Review by pgreze — APPROVED** [review]
ありがとう！

**Pull request merged**